source/11_tools_debuggers.rst

```diff
@@ -192,10 +192,11 @@ Type Checking
 .. code-block:: bash
 
 	parser.c:53:31: warning: incompatible pointer types passing
-	'struct token *' to parameter of type 'struct token_list *'
+	'struct token *' to parameter of type 'struct token_list'*
```

Comment:
Should this be 'struct token_list*' ? With the asterisk inside the single-quote?